docs/GCSCopy-action.md

```diff
@@ -26,7 +26,7 @@ It can be found on the Dashboard in the Google Cloud Platform Console.
 
 **Source Path**: Path to a source object or directory.
 
-**Destination Path**: Path to the destination. The bucket must already exist.
+**Destination Path**: Path to the destination.
```

Comment:
this is a change in behavior. Technically it is a backwards incompatible change, as somebody could have been relying on this behavior in previous versions (though this seems pretty unlikely).

@bdmogal is this intended?

Also commented on the code below, but this means an extra check is need to see if the bucket already exists. If that requires broader permissions than used to be needed, it is also a backwards incompatible change.

Reply:
@albertshau so long as the bucket is created only if it doesn't already exist (per your comment that check should be added), it should be fine, right?


We made this change so the behavior across the GCS plugins is exactly the same.

Reply:
It's a change in behavior, so it's possible somebody was relying on this behavior and expecting their pipeline to fail if the bucket doesn't exist. However, this seems unlikely, so I'm ok with changing it.

The bigger concern was about whether this is change means the plugin now needs additional permissions. @itsankit-google  I didn't see an answer on whether the existing plugin is already checking bucket existence or not.

Reply:
Yeah, so basically that was my point that the existing plugin is already checking bucket existence in StorageClient class. Here is the link to it, this method is used to perform the actual action of copy and move.
https://github.com/data-integrations/google-cloud/blob/develop/src/main/java/io/cdap/plugin/gcp/gcs/StorageClient.java#L160

Reply:
ok thanks, then there is no issue.